enterprise/api_client/ecommerce.py

```diff
@@ -81,30 +81,30 @@ def create_manual_enrollment_orders(self, enrollments):
         Since `student.CourseEnrollment` lives in LMS, we're just passing around dicts of the relevant information.
         """
         try:
-            order_repsonse = self.client.manual_course_enrollment_order.post(
+            order_response = self.client.manual_course_enrollment_order.post(
                 {
                     "enrollments": enrollments
                 }
             )
+            order_creations = order_response["orders"]
```

Comment:
Prevents a "Local variable referenced before assignment" error from being thrown

Reply:
🙏

Reply:
i'm assuming this change is unrelated functionally to the rest of the changes? just want to confirm :-)

Reply:
yeah, just preventing an error I encountered in local testing